packages/demo/src/components/examples/TableHOCServerExamples.tsx

```diff
@@ -194,20 +197,27 @@ const fetchData = (): IThunkAction => (dispatch: IDispatch, getState: () => IRea
     _.each(compositeState.predicates, (predicate: {id: string; value: string}) => {
         params[predicate.id] = predicate.value;
     });
-    return $.get('https://jsonplaceholder.typicode.com/users', params).then((response: any[], status, request) => {
-        const count = request.getResponseHeader('x-total-count');
-        const users = _.map(response, (user: any) => ({
+
+    const query = UrlUtils.toQueryString(params);
+
+    try {
+        const res = await fetch(`https://jsonplaceholder.typicode.com/users?${query}`);
+        const count = (res.headers.has('x-total-count') && res.headers.get('x-total-count')) || null;
+        const data = await res.json();
+
+        const users = data.map((user: any) => ({
             city: user.address.city,
             username: user.username,
             email: user.email,
-            dateOfBirth: moment().subtract(user.address.city.length, 'years').toDate(), // fake a year of birth
+            dateOfBirth: moment('1995-12-25').toDate(), // fake a year of birth
```

Comment:
I don't think you need to change that?
The date picker in the example is used to filter on this dateOfBirth